azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/diagnostic_logger.py

```diff
@@ -0,0 +1,87 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+from os import makedirs
+from os.path import exists, join
+
+from azure.monitor.opentelemetry.distro.constants import (
+    _CUSTOMER_IKEY,
+    _DIAGNOSTIC_LOG_PATH,
+    _EXTENSION_VERSION,
+    _IS_ON_APP_SERVICE,
+    _env_var_or_default,
+)
+from azure.monitor.opentelemetry.distro.version import VERSION
+
+_OPENTELEMETRY_DIAGNOSTIC_LOGGER_NAME = "opentelemetry"
+_DISTRO_DIAGNOSTIC_LOGGER_NAME = "azure.monitor.opentelemetry.distro"
```

Comment:
```suggestion
_DISTRO_DIAGNOSTIC_LOGGER_NAME = "azure.monitor.opentelemetry.distro.diagnostic_logger"
```

To separate it from the regular logging hierarchy.

Reply:
My thought here was that it would also capture logs from elsewhere in the distro, but I guess that's unnecessary since we have the helper methods.

Reply:
I'll make the change manually since it will likely change a couple tests.

Reply:
Done